controls/radvirtualgrid/mvvm.md

```diff
@@ -0,0 +1,327 @@
+	        {
+	            int propertyIndex = this.GetPropertyIndex("Name");
+	            if (columnIndex == propertyIndex)
+	            {
+	                return true;
+	            }
+	
+	            return base.IsColumnReadOnly(columnIndex);
+	        }
+	
+	        /// <summary>
+	        /// Gets the index of a given property
+	        /// </summary>
+	        /// <param name="propertyName">The property name</param>
+	        /// <returns>The property index</returns>
+	        private int GetPropertyIndex(string propertyName)
+	        {
+	            ItemPropertyInfo itemProperty = this.ItemProperties.Where(ip => ip.Name == propertyName).FirstOrDefault();
+	            int index = this.ItemProperties.IndexOf(itemProperty);
+	
+	            return index;
+	        }
+	
+	        /// <summary>
+	        /// Persists the selected indexes
+	        /// </summary>
+	        /// <param name="selectedIndexes"></param>
+	        private void PersistSelection(IEnumerable<int> selectedIndexes)
+	        {
+	            foreach (int index in selectedIndexes)
+	            {
+	                this.selectedIndexes.Add(index);
+	            }
+	        }
+	
+	        /// <summary>
+	        /// Adds the previously persisted indexes
+	        /// </summary>
+	        /// <param name="selectedIndexes"></param>
+	        private void UpdateSelection(IEnumerable<int> selectedIndexes)
+	        {
+	            foreach (int index in selectedIndexes)
+	            {
+	                if (!this.ParentGrid.SelectedIndexes.Contains(index))
+	                {
+	                    this.ParentGrid.ToggleIndexSelection(index);
+	                }
+	            }
+	        }
+    }
+{{endregion}}
+
+## Defining the View Model
+
+Furthermore, the custom DataProvider needs to be exposed by a view model.
+
+#### __[C#] Example 2: Defining the View Model__
+
+{{region radvirtualgrid-mvvm_1}}
+	public class MyViewModel: ViewModelBase
+    {
+        private MyDataProvider dataProvider;
+
+        public MyDataProvider DataProvider 
+        {
+            get 
+            {
+                if (this.dataProvider == null)
+                {
+                    this.dataProvider = new MyDataProvider(Club.GetClubs());
+                }
+
+                return this.dataProvider;
+            }
+        }
+    }
+{{endregion}}
+
+## Populate RadVirtualGrid
+
+Finally, the View Model is set to be the DataContext of __RadVirtualGrid__ and its __DataProvider__ property is bound to the custom DataProvider.
+
+#### __[XAML] Example 3: Populating RadVirtualGrid__
+
+{{region radvirtualgrid-mvvm_2}}
+	<Window.Resources>
+        <local:MyViewModel x:Key="MyViewModel"/>
+    </Window.Resources>
+    <Grid DataContext="{StaticResource MyViewModel}">
+        <telerik:RadVirtualGrid DataProvider="{Binding DataProvider}" 
+                                Margin="5" >
+        </telerik:RadVirtualGrid>
+    </Grid>
+{{endregion}}
+
+## See Also
```

Comment:
Don't leave spaces between the links in the See Also section. They take too many space. You can place each on a new row. Here is the comparison:

* [Getting Started]({%slug virtualgrid-getting-started2%})

* [Custom DataProvider]({%slug virtualgrid-custom-dataprovider%})

* [Editing]({%slug virtualgrid-editing%})

VS

* [Getting Started]({%slug virtualgrid-getting-started2%})
* [Custom DataProvider]({%slug virtualgrid-custom-dataprovider%})
* [Editing]({%slug virtualgrid-editing%})